### DISCUSSION TITLE
对于个人部署，目前服务器端数据库版本的登录过于臃肿

### DISCUSSION THREAD

**Discussion by sherlcok314159** [lobehub/lobehub#4584] [Self Hosting] [answered]
### 📦 部署环境

Docker

### 📌 软件版本

latest

### 💻 系统环境

Ubuntu

### 🌐 浏览器

Firefox

### 🐛 问题描述

首先，很感谢作者开发出服务器端数据库的版本，这也是我一直期待看到的功能。

作者也花了诸多心血用于各种身份验证系统，但是有一点需求可能被忽略了，对于个人 self-hosted 用户而言，我并不希望进行如此复杂的身份验证，也正如很多自部署应用的方式：用户名+密码足矣，犯不着再去开个实例

也许是目前的版本中已经可以实现我说的这种，但目前在 doc 中尚未找到，如有遗漏，请纠正我

总结：对于个人用户而言，可以只需要设置账号，密码这种简单 / 甚至没有 的模式，而不是通过第三方来进行身份验证

[a comment by an automated account is not shown]

**Comment by arvinxx** [comment, marked as answer]
不计划做。在你看来简单的用户名密码的功能背后的复杂度基本上和写一个 LobeChat 差不多，我并不想承担这种复杂度，专业的事情交给更专业的团队去做更合适。

另外加一个auth组件是否臃肿这个完全因人而异，有这个感受大概率是因为之前可能你没玩过类似的部署应用。有的用户还因为部署 casdoor 之后觉得好用，然后把他所有的服务都统一到一个里。

既然玩自部署了，这个复杂度还是需要你自己接下的

[a comment by an automated account is not shown]

**Comment by arvinxx** [comment]
给你看一个 OpenWebUI 自己做了登录注册，并默认给你预置一个账号密码的后果：

<img width="1052" alt="image" src="https://github.com/user-attachments/assets/bd8cf2b7-1cda-4ca2-9fe3-880bb5c59293">

**安全无小事**

**Reply by Zoumachuan** [reply]
哈人
